src/artemis/devices/zebra.py

```diff
@@ -0,0 +1,276 @@
+from __future__ import annotations
+from typing import Iterator, List, Dict
+from enum import Enum
+
+from ophyd import Component, Device, EpicsSignal, StatusBase
+from ophyd.status import SubscriptionStatus
+
+from functools import partial, partialmethod
+
+PC_ARM_SOURCE_SOFT = 0
+PC_ARM_SOURCE_EXT = 1
+
+PC_GATE_SOURCE_POSITION = 0
+PC_GATE_SOURCE_TIME = 1
+PC_GATE_SOURCE_EXTERNAL = 2
+
+PC_PULSE_SOURCE_POSITION = 0
+PC_PULSE_SOURCE_TIME = 1
+PC_PULSE_SOURCE_EXTERNAL = 2
+
+# Sources
+DISCONNECT = 0
+IN1_TTL = 1
+IN2_TTL = 4
+PC_ARM = 29
+PC_GATE = 30
+PC_PULSE = 31
+AND3 = 34
+AND4 = 35
+OR1 = 36
+PULSE1 = 52
+SOFT_IN3 = 62
+
+# Instrument specific
+TTL_DETECTOR = 2
+TTL_XSPRESS3 = 3
+TTL_SHUTTER = 4
+
+
+def epics_signal_put_wait(pv_name: str, wait: str = 1.0) -> EpicsSignal:
```

Comment:
Would be useful for this method to be outside of the zebra module so it can be used for all devices.

Reply:
Agreed, done!